lib/game/game_setup.ex

```diff
@@ -24,11 +24,20 @@ defmodule Game.GameSetup do
     Players.set_players(player_one, player_two)
   end
 
-  @spec select_player_type(String.t(), list()) :: player
-  defp select_player_type(player_selection, player_types) do
-    {selection_number, _selection_decimal} = Integer.parse(player_selection)
-    %{type: player_type} = Enum.at(player_types, selection_number - 1)
-    player_type
+  @spec select_player_type(String.t(), any()) :: player
+  defp select_player_type(player_symbol, interface) do
```

Comment:
`interface` is confusing to me here. I don't remember what it is.

Reply:
`interface` is the struct that encapsulates both the command line wrapper itself and the command line formatter 😄

Reply:
Thanks for clarifying! so it's really display or view related. Is it possible that we name it something more specific? for example, can we call this user_interface?

Reply:
Yeah, that sounds like a good idea to me, I like the name `user_interface` 😄